### ISSUE TITLE
[KEYBOARD]: GIBCT EYB - Skip to your estimated benefits link MUST work in all browsers

### ISSUE THREAD

**Issue by 1Copenut** [department-of-veterans-affairs/va.gov-team#11307]
## [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

The skip to your estimated benefits functionality is not working in IE11. I believe this has to do with the skip being a button instead of an in-page link. Screen shot and code snippet attached below.

## Point of Contact

**VFS Point of Contact:** _Trevor_
## Acceptance Criteria

- [ ] Pressing the skip link should behave consistently in IE11 as it does with all other browsers
- [ ] No regression in other browsers. This includes Chrome, Firefox, Safari, for MacOS, and Chrome, Firefox, Edge for Windows 10. Should also include iOS devices and Android devices if available.

## Environment

* Windows 10
* Internet Explorer 11
* https://staging.va.gov/gi-bill-comparison-tool/profile/31801227

## Steps to Recreate

1. Enter the URL above or another URL for GIBCT profile page
1. Tab through the page until the first Skip to your estimated benefits link has keyboard focus
1. Press `ENTER` to activate the link
1. Press `TAB` again and verify that focus is not going to the correct place (inside the Estimated benefits gray panel) in IE11

## Possible Fixes (optional)

I believe the fix is to swap the button for an anchor link:

```diff
<div class="vads-u-padding-bottom--2p5">
- <button
-   aria-label="Skip to your estimated benefits"
-   class="va-button-link learn-more-button eyb-skip-link"
-   type="button"
- >Skip to your estimated benefits</button>
+ <a
+   class="va-button-link learn-more-button eyb-skip-link"
+   href="#estimated-benefits"
+ >Skip to your estimated benefits</a>
</div>
```

## WCAG or Vendor Guidance (optional)

* [Keyboard: Understanding SC 2.1.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-keyboard-operable.html)

## Screenshots or Trace Logs

![Screen Shot 2020-07-16 at 10 24 47 AM](https://user-images.githubusercontent.com/934879/87691918-c551b980-c750-11ea-80a1-cfc7d94ac463.png)

**Comment by elder-joseph** [comment]
Associated PR: https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/vets-website/14047

**Comment by sonntag-adam** [comment]
@1Copenut this is in staging ready for review

**Comment by 1Copenut** [comment]
Verified this is working in IE11 as expected. Moving this issue to close.
